generated_api_shadow/BUILD

```diff
@@ -145,7 +145,6 @@ proto_library(
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
         "//envoy/config/retry/omit_canary_hosts/v2:pkg",
         "//envoy/config/retry/previous_hosts/v2:pkg",
-        "//envoy/config/retry/previous_hosts/v3:pkg",
```

Comment:
@htuch not sure why this is necessary - seems to be unrelated to this PR as im seeing similar on others